lib/util/src/platform.c

```diff
@@ -220,10 +216,7 @@ hse_platform_init(void)
 
     dt_init();
     event_counter_init();
-
-    err = logging_init(&hse_gparams.gp_logging);
-    if (err)
-        return err;
+    vlb_init();
```

Comment:
Don't move this, it prevents using anything initialized by hse_cpu_init() or in vlb_init().
Probably hse_cpu_init() should be called first of all these...

Reply:
I will move hse_cpu_init() up then.

Reply:
Oops, I meant hse_cpu_init() and timer_init()...   We're not not using timers in vlb_init() right now, but I don't wish to preclude it...